rust/test_service_elections/src/lib.rs

```diff
@@ -71,6 +72,31 @@ mod service {
         }
     }
 
+    // TODO: create a macro for external records
```

Comment:
The macro could expand to something like the "Inside the Service Module" section of https://doc-sys.psibase.io/rust-service/tables.html#storing-structs-defined-elsewhere